src/tests/blueprint/t_blueprint_mpi_mesh_transform.cpp

```diff
@@ -927,9 +927,9 @@ TEST(conduit_blueprint_mesh_examples, generate_corners_wonky)
     }
     test_adjset_points(mesh, "corner_pairwise_adjset");
 
-    conduit::Node info;
+    conduit::Node info, opts;
     bool same_pointwise = conduit::blueprint::mpi::mesh::utils::adjset::compare_pointwise(
-                              mesh, "corner_pairwise_adjset", info, MPI_COMM_WORLD);
+                              mesh, "corner_pairwise_adjset", opts, info, MPI_COMM_WORLD);
```

Comment:
do opts need to be mandatory for `compare_pointwise`?

Reply:
Again, removed it in favor of default options.